src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int INDEX_OF_BUSINESS_TYPE_PARAMETER = 0;
+    private static final int INDEX_OF_AMOUNT = 1;
+    private static final int INDEX_OF_SUPPLY_PARAMETER = 0;
+    private static final int INDEX_OF_BUY_PARAMETER = 1;
+    private static final String COMA_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] receivedData = readDataFromFile(fromFileName);
+        String[] resultParameters = dataProcessing(receivedData);
+        int supplyAmount = Integer.parseInt(resultParameters[INDEX_OF_SUPPLY_PARAMETER]);
+        int buyAmount = Integer.parseInt(resultParameters[INDEX_OF_BUY_PARAMETER]);
+        String report = createReport(supplyAmount, buyAmount);
```

Comment:
Let's pass String[] resultParameters to createReport() and replace logic 
```
int supplyAmount = Integer.parseInt(resultParameters[INDEX_OF_SUPPLY_PARAMETER]);
int buyAmount = Integer.parseInt(resultParameters[INDEX_OF_BUY_PARAMETER]);
```

to createReport method